src/webgl/p5.RendererGL.js

```diff
@@ -2545,4 +2557,24 @@ p5.prototype._assert3d = function (name) {
 
 p5.RendererGL.prototype.tessyVertexSize = 12;
 
+p5.RendererGL.prototype.background = function (...args) {
+  if (args.length === 1 && args[0] instanceof p5.Image) {
+    const img = args[0];
+    this.clear(); // clear previous frame
+    this._pInst.push();
+    this._renderer._setDefaultCamera(); // reset camera
```

Comment:
Since `this` is already a renderer, does `this._renderer` here work, or does it need to just be `this._setDefaultCamera`?